trulens_eval/trulens_eval/feedback/provider/hugs.py

```diff
@@ -173,6 +174,48 @@ def get_scores(text):
 
         return l1, dict(text1_scores=scores1, text2_scores=scores2)
 
+    @_tci
+    def context_relevance(self, prompt: str, context: str) -> float:
+        """
+        Uses Huggingface's truera/context_relevance model, a
+        model that uses computes the relevance of a given context to the prompt. 
+        
+        **Usage:**
+        ```python
+        from trulens_eval import Feedback
+        from trulens_eval.feedback.provider.hugs import Huggingface
+        huggingface_provider = Huggingface()
+
+        feedback = Feedback(huggingface_provider.context_relevance).on_input_output() 
```

Comment:
can you change to use .on_input().on(context)

Reply:
You mean the new context selectors?